lib/command_graph.c

```diff
@@ -0,0 +1,480 @@
+/*
```

Comment:
Not sure what you mean with the copyright header?

Can ofc reformat the code to kernel style.